sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb

```diff
@@ -0,0 +1,1094 @@
+{
```

Comment:
<p>These warnings are repeated 2-3 times throughout the notebook. Is that intentional, or can they be stated only one time?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
<p>Yeah .. thanks for pointing this out. There are 4 places. The one under <strong><em>Install the </em></strong><code><strong><em>smjsindustry</em></strong></code><strong><em> library </em> </strong>section i think can be removed. Please it is not related directly to SEC EDGAR database. </p><p> </p><p>And the other three, the first one is for not financial advice warning; the second one is for SEC EDGAR database warning; The third one is to summarize the two warnings in the end. So i think we can keep them</p>